Realm/Tests/RealmTests.mm

```diff
@@ -175,6 +177,58 @@ - (void)testFileFormatUpgradeRequiredButDisabled {
     RLMAssertThrowsWithCodeMatching([RLMRealm realmWithConfiguration:config error:nil], RLMErrorFileFormatUpgradeRequired);
 }
 
+- (void)testExceedingVirtualAddressSpace {
```

Comment:
For me, takes 2 seconds for OS X/64-bit sim, 200 ms for 32-bit sim, and 500 ms on a 64-bit device. IMO that's long enough to justify making it device/32-bit sim only.